### ISSUE TITLE
[Library] No media Information when playing exported media

### ISSUE THREAD

**Issue by tonyschuite** [asciidisco/plugin.video.netflix#167]
Exported series and movies are imported into the kodi library fine. However because of the plugin call in the strm files, no media information is shown in the OSD. this also results in that other plugins are not using the info (for example Trakt).
When playing from within the plugin, everything works fine

my solution would be to add the info to the plugin call within the strm files.
plugin://plugin.video.netflix/?action=play_video&video_id=80117471&info_labels=XXXX

in Navigation.py on line 128 there is already a parser for this command (I think). My knowledge of the code is too limited to be of any further help.

**Comment by asciidisco** [comment]
Bringing in @hawkeyexp as I believe he added this (in some sort or another) already.

**Comment by hawkeyexp** [comment]
@asciidisco you are right - it was merged on sep 20 "Enable metacaching"

@tonyschuite
It should work since v0.11.25. If the export is done with an older version the metadata is missing in the cache. The safest way is to remove it from export and reexport it.

**Comment by asciidisco** [comment]
@hawkeyexp Thanks for the info.

@tonyschuite I´m closing this, if a re-export doesn't lead to the expected behaviour, feel free to reopen :)

**Comment by tonyschuite** [comment]
Yes, I do have slightly more info now. However no library info. I think this has to do with the limitation of the strm files / plugin reference. So, I don't think it can be done.

**Comment by hawkeyexp** [comment]
So you get the mediainfo from netflix now with a image in landscape format and a short plot ? Can you post an image ? If This is correct pr ##170 will add this option and grab the content directly from kodi if available.

**Comment by tonyschuite** [comment]
update: this only works when you have played a video in the season from within the netflix addon

i guess he only collects the metadata then...

**Comment by tonyschuite** [comment]
@hawkeyexp yes, i do! but only if I have played an episode from within the netflix addon. If I export from the addon and not watch it from there, I don't get the extra metadata. 

basically I need to open one episode of each season in the netflix addon to get the metadata correct. #170 should be a better alternative I think. would love to test it.

**Comment by hawkeyexp** [comment]
I have to correct the info of asciidisco:
the episode list must be only shown one time at the moment inside the netflix addon than metadata is cached and visible on playback.You do not need to play every epsode :-). PR #170 is based on it to get episode number etc. but this part of the addon will get some more love later.

**Comment by joaosagrath** [comment]
I have this same issue. But, I did some tests and his the result:

![screenshot001](https://user-images.githubusercontent.com/16491360/31395015-198f9452-adb6-11e7-9401-a467beb5cd93.png)
Screenshot 01: If I go to addon, and just export the serie/movie to library, and after that, go to library and play the strm episode/movie, I don't get the metadata

![screenshot002](https://user-images.githubusercontent.com/16491360/31395087-4de82cf0-adb6-11e7-87dd-833e9b18779f.png)
Screenshot 02: If I go to the item that I exported, in the addon and play, I get the metadata (landscape image and plot)

![screenshot003](https://user-images.githubusercontent.com/16491360/31395181-8a46ba36-adb6-11e7-9fcc-b90623e57b61.png)
But If I return to my library, and play again the strm episode/movie, I get the metadata, but the landscape is different, and the plot is the same.

Hope this helps!
Cheers
